### PR TITLE
Update getting-started-part-1.md

### PR DESCRIPTION
**Pull request by tassoman**
Adding a note on plugin's naming convention on the marketplace

### CODE REVIEW

File: docs/4.x/getting-started-part-1.md [hunk 1]
```diff
@@ -186,7 +186,11 @@ The configured DB user for the tests should also have privileges to create and d
 
 ## Create a plugin
 
-Your development environment is set up, and you are now ready to create a plugin! Creating a plugin consists primarily of creating a couple of files which can be done by running the following command:
+Your development environment is set up, and you are now ready to create a plugin! 
+
+First of all, you need to choose the plugin's name. Remember, if you're going to publish on [Matomo's marketplace](https://plugins.matomo.org/), CamelCase will be splitted into words. So, the name «MyPlugin» will be published as «My Plugin».
```
Comment (Findus23) [inline docs/4.x/getting-started-part-1.md]:
```suggestion
First of all, you need to choose the plugin's name. Remember, if you're going to publish on [Matomo's marketplace](https://plugins.matomo.org/), CamelCase will be split into words. So, the name "MyPlugin" will be published as "My Plugin".
```

I think "normal" quotation marks fit better with the rest of the text.

**Review by Findus23 — CHANGES_REQUESTED** [review]
Just a pedantic nitpick...

**Comment by tassoman** [comment]
I added a typofix commit :sweat_smile:

**Comment by tsteur** [comment]
Thanks @tassoman

**Pull request merged**